### PR TITLE
Redeploy-certificates will fail for registry and router if user is not system:admin

### PR DESCRIPTION
**Pull request by jkaurredhat**
Redeploy-certificates will fail for registry and router if user is not system:admin

Signed-off-by: jkaurredhat <jkaur@redhat.com>

### CODE REVIEW

File: playbooks/common/openshift-cluster/redeploy-certificates/router.yml [hunk 1]
```diff
@@ -116,6 +116,7 @@
         tls.crt="{{ mktemp.stdout }}/openshift-hosted-router-certificate.pem"
         tls.key="{{ mktemp.stdout }}/openshift-hosted-router-certificate.key"
         --type=kubernetes.io/tls
+        --config={{ mktemp.stdout }}/admin.kubeconfig
         --confirm
         -o json | {{ openshift.common.client_binary }} replace -f -
```
Comment (abutcher) [inline playbooks/common/openshift-cluster/redeploy-certificates/router.yml]: I think we'll also need to pass the config after the pipe to `oc replace`.

**Comment by sdodson** [comment]
Lets make sure we check all instances for this so we're relatively sure that we've addressed this once and for all.

**Comment by jkaurredhat** [comment]
Thanks for the review. I actually missed it. 

I have made the required changes now.

**Comment by sdodson** [comment]
aos-ci-test

**Comment by sdodson** [comment]
[merge][severity:blocker]

**Pull request merged**